include/quicr/detail/messages.h

```diff
@@ -59,6 +59,10 @@ namespace quicr::messages {
 
         GOAWAY = 0x10,
 
+        SUBSCRIBE_ANNOUNCES = 0x11,
```

Comment:
All of these need to be style formatted. For example `kSubscribeAnnounces`

Reply:
yes I saw the incosistency. I feel that can be done in a follow up PR.